openapi-cli/src/test/java/io/ballerina/openapi/generators/openapi/NegativeConstraintTests.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Lets update the license header. Check for other places as well.
```suggestion
 * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```